docs/_data/prague-2020-schedule.yaml

```diff
@@ -0,0 +1,77 @@
+writing_day:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:30'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:45'
+    title: Team up and start working
+  - time: '12:30'
+    title: Lunch Break
+  - time: '5:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:00'
```

Comment:
Normally we've started Prague at 10am, did we change this on purpose?

Reply:
NOPE @plaindocs we need to shift everything forward by 1hr <3